nx-X11/programs/Xserver/composite/compalloc.c

```diff
@@ -481,14 +633,21 @@ compAllocPixmap (WindowPtr pWin)
 	DamageRegister (&pWin->drawable, cw->damage);
 	cw->damageRegistered = TRUE;
     }
+
+    /* Make sure our borderClip is up to date */
+    REGION_UNINIT(pWin->drawable.pScreen, &cw->borderClip);
+    REGION_COPY(pWin->drawable.pScreen, &cw->borderClip, &pWin->borderClip);
```

Comment:
REGION_UNINIT and REGION_COPY used to be macros, but are now inline functions (RegionUninit, RegionCopy). See https://github.com/ArcticaProject/nx-libs/commit/4a8d6eca395e1ec0490e20e7971177f688da18ec for details.

The old macro implementation has been kept as backwards compat mech, but we should really use the inline function calls here, I'd say.